src/scss/pattern-library-compatibility/content-card.scss

```diff
@@ -29,8 +33,8 @@
 	line-height: 1.5;
 }
 
-.content-card[data-sidebar-card]:not(:last-child) {
-	margin-bottom: 16px;
+.content-card + .content-card[data-sidebar-card] {
```

Comment:
Is there a page where a content card and a content card meant for the sidebar are actually next to each other (`+` selector)? Our current sidebar cards are in an `<aside>` element.